src/components/TextBoxField/index.tsx

```diff
@@ -177,22 +169,21 @@ const TextBoxField = forwardRef(
         error={error}
         id={id}
         label={label}
-        max={max}
+        size={size}
         maxLength={maxLength}
-        min={min}
         minLength={minLength}
         multiline={multiline}
         name={input.name}
         notice={notice}
-        onBlur={(event: FocusEvent<HTMLInputElement>) => {
+        onBlur={(event: FocusEvent<HTMLTextAreaElement>) => {
```

Comment:
HTMLInputElement wasn't working ?

Reply:
+1 Or at least it should be `<HTMLInputElement | HTMLTextAreaElement>`